src/vcpkg/binarycaching.cpp

```diff
@@ -1160,6 +1306,36 @@ namespace
                 handle_readwrite(
                     state->url_templates_to_get, state->azblob_templates_to_put, std::move(p), segments, 3);
             }
+            else if (segments[0].second == "x-gcs")
+            {
+                // Scheme: x-gsutil,<prefix>[,<readwrite>]
```

Comment:
```suggestion
                // Scheme: x-gcs,<prefix>[,<readwrite>]
```
Unless you'd prefer to rename it everywhere to `x-gsutil`